app/controllers/reports/regions_controller.rb

```diff
@@ -36,6 +41,15 @@ def details
     @quarterly_registrations = @data[:quarterly_registrations]
     @top_region_benchmarks = @data[:top_region_benchmarks]
     @last_registration_value = @data[:registrations].values&.last || 0
+
+    unless @region.is_a?(FacilityGroup)
```

Comment:
Is it going to be shown in pages other than `Facility`-level reports? If not, then should we just make this 

```ruby
if @region.is_a?(Facility)
```

Reply:
Good call @kitallis. Fixed!